.devcontainer/launch.sh

```diff
@@ -11,17 +11,38 @@ print_help() {
     echo "the top-level devcontainer in .devcontainer/devcontainer.json will be used."
     echo ""
     echo "Options:"
-    echo "  -c, --cuda       Specify the CUDA version. E.g., 12.2"
-    echo "  -H, --host       Specify the host compiler. E.g., gcc12"
-    echo "  -d, --docker     Launch the development environment in Docker directly without using VSCode."
-    echo "  -h, --help       Display this help message and exit."
+    echo "  -c, --cuda               Specify the CUDA version. E.g., 12.2"
```

Comment:
I'm skipping the devcontainer/launch/entrypoint stuff, assuming they're the same as #1779.